controllers/ocsinitialization/sccs.go

```diff
@@ -176,38 +175,3 @@ func newNooBaaEndpointSCC(namespace string) *secv1.SecurityContextConstraints {
 
 	return scc
 }
-
-func newMetricsExporterSCC(namespace string) *secv1.SecurityContextConstraints {
```

Comment:
@umangachapagain why SCC is removed? Dont we need SCC for metrics exporter?

Reply:
Not anymore. Earlier we needed it for some extra permissions, but now it's working without those.